README.md

```diff
@@ -81,7 +66,7 @@ The `storage_options` can be instantiated with a variety of keyword arguments de
    anonymous access will not be attempted. Otherwise the value for `anon` resolves to True.
 - `location_mode`: valid values are "primary" or "secondary" and apply to RA-GRS accounts
 
-For more argument details see all arguments for [`AzureBlobFileSystem` here](https://github.com/fsspec/adlfs/blob/f15c37a43afd87a04f01b61cd90294dd57181e1d/adlfs/spec.py#L328) and [`AzureDatalakeFileSystem` here](https://github.com/fsspec/adlfs/blob/f15c37a43afd87a04f01b61cd90294dd57181e1d/adlfs/spec.py#L69).
+For more argument details see all arguments for [`AzureBlobFileSystem` here](https://github.com/fsspec/adlfs/blob/f15c37a43afd87a04f01b61cd90294dd57181e1d/adlfs/spec.py#L328) 
```

Comment:
Instead of linking to the code, let's instead link to the rendered HTML docs: https://fsspec.github.io/adlfs/api/#adlfs.AzureBlobFileSystem. Mainly, there is a hardcoded commit SHA here so if arguments are added in the future, this link will drift away from what is in main.